app/services/profile_fields/field_definition.rb

```diff
@@ -20,14 +20,16 @@ def fields
 
       private
 
-      def field(label, input_type, placeholder: nil, description: nil, group: @group, display_area: "left_sidebar")
+      def field(label, input_type, placeholder: nil, description: nil, group: @group, display_area: "left_sidebar",
```

Comment:
This signature is getting a bit unwieldy. I wonder if we should do the following:

```ruby
def field(label, input_type, **attributes)
  attributes = attributes.reverse_merge(group: @group, display_area: "left_sidebar")
  fields << {
    label: label,
    input_type: input_type,
    placeholder_text: attributes[placeholder],
    ...
```

This is a pattern that Rails uses internally too.

Reply:
TIL - Done ✅